auth_service/config.py

```diff
@@ -62,6 +62,8 @@ class Config(ApiConfigBase):
     service_name: str = "auth_service"
     log_level: LogLevel = "debug"
     run_auth_adapter: bool = False
+    auth_int_keys: Optional[str] = None
```

Comment:
Is this a pair of secret and public key? If so, shouldn't we inject as separate parameters? Would be much easier to store them separately in vault.

Reply:
They are stored as a JWK structure, so it's easier to have them merged.